linkerd/app/core/src/errors.rs

```diff
@@ -1,4 +1,4 @@
-use http::{header::HeaderValue, StatusCode};
+use http::header::HeaderValue;
```

Comment:
Why this change? Why not keep `StatusCode` in scope?

Reply:
We are primarily using `http::StatusCode` everywhere else in the file and I had made an intermediate change where this became unused. I'll change all the other references to it and add the import back.